client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -36,6 +37,25 @@ const BlockList = ( props: BlockListProps ) => <OriginalBlockList { ...props } /
 export function Gutenboard() {
 	const { __: NO__ } = useI18n();
 
+	// TODO: Explore alternatives for loading fonts and optimizations
+	// TODO: Don't load like this
```

Comment:
Wow, loading all the fonts at one go is rather heavy indeed. I think fastest way is to have the font pairings data come with come with manually created SVG thumbnails.

Automatic generating of thumbnails or... this sounds adventurous.. extracting glyphs from different fonts and put them into a single font seems like a lot of work.